app/controllers/projects_controller.rb

```diff
@@ -30,6 +30,39 @@ def submit_payment
     @project = Project.find(params[:project_id])
   end
 
+  def pay
+    @project = Project.find(params[:project_id])
+    if @project.developers.any? {|dev| dev.status == 'unavailable'}
+      flash[:error] = "Some of your developers are unavailable"
+    else
+      @project.update_attribute(:status, 1)
+      @project.developers.each do |dev|
+        dev.update_attribute(:status,'unavailable')
+      end
+    end
+    redirect_to contractor_project_path(@project)
+  end
+
+  def complete
+    @project = Project.find(params[:project_id])
+    @project.update_attribute(:status, 2)
+    @project.developers.each do |dev|
+      dev.update_attribute(:status,'available')
+    end
+    flash[:sucess] = "Project Complete - Congratulations!"
+    redirect_to contractor_project_path(@project)
+  end
+
+  def cancel
+    @project = Project.find(params[:project_id])
+    @project.update_attribute(:status, 3)
+    @project.developers.each do |dev|
+      dev.update_attribute(:status,'available')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.